docs/atlascli/command/atlas-clusters-update.txt

```diff
@@ -16,8 +16,6 @@ Modify the settings of the specified cluster.
 
 You can specify modifications in a JSON configuration file with the --file flag.
 		
-You can modify only M10 or larger clusters that are single-region replica sets.
```

Comment:
you can't use this command for shared tier, ?I think that was the idea on this line